cost-analyzer/values.yaml

```diff
@@ -706,12 +706,24 @@ awsstore:
   useAwsStore: false
   createServiceAccount: false
 
+federatedETL:
+  federatedCluster: false
+  primaryCluster: false
+  redirectS3Backup: false
```

Comment:
is this config + the next critical? feels like it potentially adds unnecessary complexity for users + our support efforts

Reply:
The `redirectS3Backup`? This was an ask to allow people to migrate their existing Thanos federated data to the federator. Since it's part of federated ETL, I elected to put it here.